Makefile

```diff
@@ -7,7 +7,7 @@ all: website favicon test
 favicon:
 	node lib/badge-cli.js '' '' '#bada55' .png > favicon.png
```

Comment:
Is it just me, or does this no longer produce anything?
Also without the `.png` it produces this for me:
![](https://sendeyo.com/up/d/65022234c5)

Reply:
Huh, weird. `node lib/badge-cli.js '' '' '#bada55' > favicon.svg` produces the right svg for me. No quotes… wonder if that could be a windows thing? Is it better if `'' ''` is replaced with `"" ""` in the makefile?

However the png is not working for me either. It makes a valid PNG but it's all gray. I upgraded imagemagick, and it changed to a different gray (!) but still isn't working correctly.

Reply:
Opened a new issue: #1788